t/data/tests/tests/pre_boot.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2019 SUSE LLC
```

Comment:
This PR has really lying around for a while now; you could update the year. The same counts for the other occurrences.

Reply:
done